src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -0,0 +1,712 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Static Analysis Results Format (SARIF) Version 1.0 JSON Schema (Draft 1.0.0-beta.1)",
+  "description": "Static Analysis Results Format (SARIF) Version 1.0 JSON Schema (Draft 1.0.0-beta.1). SARIF defines a standard format for the output of static analysis tools.",
+  "additionalProperties": false,
+  "type": "object",
+  "properties": {
+    "version": 
+    {
+      "description": "The SARIF tool format version of this log file. This value should be set to 1.0.0-beta.1, currently. This is the third proposed revision of a file format that is not yet completely finalized.",
+      "enum": [ "1.0.0-beta.1"]
```

Comment:
should this now be beta.2? we agreed to version with each breaking change, yes? we should definitely version if beta.1 ships on nugget right now.